src/componentsNext/Pagination/__tests__/describePagination.test.ts

```diff
@@ -0,0 +1,137 @@
+import { describe, expect, test } from "vitest";
+
+import describePagination from "../describePagination";
+
+describe("describePagination", () => {
+  describe("it describes a simple 10 page pagination", () => {
+    test("page 1 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 1 });
+      expect(pagination).toStrictEqual([1, 2, "...", 9, 10]);
+    });
+
+    test("page 2 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 2 });
+      expect(pagination).toStrictEqual([1, 2, 3, "...", 9, 10]);
+    });
+
+    test("page 5 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 5 });
+      expect(pagination).toStrictEqual([1, 2, "...", 4, 5, 6, "...", 9, 10]);
+    });
+
+    test("page 6 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 6 });
+      expect(pagination).toStrictEqual([1, 2, "...", 5, 6, 7, "...", 9, 10]);
```

Comment:
... and in this case, one could render the 8.

So it seems you need 11+ pages to get both gaps.

Reply:
I updated the tests to test everything against 11 pages to cover the use case of two ellipsis https://github.com/direktiv/direktiv-ui/pull/412/commits/e5567f8575cbef6a920f18669e0c17ba6ce66a95